### PR TITLE
support ShowError option in Pivot table

### PR DESCRIPTION
**Pull request by jinhyuk-kim-ca**
# PR Details
This is a request to enable pivot table option. ShowError

## Description
In Excel, PivotTable Options - Layout & Format - "For error values show"

## Motivation and Context
This Pivot table option allows to remove Error(#DIV/0!)

## How Has This Been Tested
New option added into test file

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: pivotTable.go [hunk 1]
```diff
@@ -34,6 +34,7 @@ type PivotTableOption struct {
 	PageOverThenDown    bool
 	MergeItem           bool
 	CompactData         bool
+	ShowError			bool
```
Comment (xuri) [inline pivotTable.go:37]: please format code by `gofmt`

File: xmlPivotTable.go [hunk 2]
```diff
@@ -31,7 +31,7 @@ type xlsxPivotTableDefinition struct {
 	DataCaption             string                   `xml:"dataCaption,attr"`
 	GrandTotalCaption       string                   `xml:"grandTotalCaption,attr,omitempty"`
 	ErrorCaption            string                   `xml:"errorCaption,attr,omitempty"`
-	ShowError               bool                     `xml:"showError,attr,omitempty"`
+	ShowError               *bool                     `xml:"showError,attr,omitempty"`
```
Comment (xuri) [inline xmlPivotTable.go:34]: If the specified pointer of boolean type for this field, the `omitempty` JSON struct tag can be removed, we can also remove it in `ShowDrill` JSON tags.

**Review by xuri — CHANGES_REQUESTED** [review]
Thanks for you PR, I'll left some comments.

File: xmlPivotTable.go (hunk 2 above)
Comment (jinhyuk-kim-ca) [reply xmlPivotTable.go]: Thanks for your review and suggestion. I've applied what you mentioned above. 
- gofmt
- remove json omitempty tag for both ShowError, ShowDrill

**Comment by xuri** [comment]
LGTM, thank you.

**Pull request merged**